### ISSUE TITLE
Downloads in http://www.saurus.info/

### ISSUE THREAD

**Issue by fgeek** [sauruscms/Saurus-CMS-Community-Edition#62]
You might want to disable downloads in http://www.saurus.info/ web page as the product is [discontinued](http://www.saurus.info/development-of-saurus-cms-officially-discontinued/). People might not notice the blog post. This is a real problem if/when someone finds new security vulnerabilities from the code.

**Comment by koosh** [comment]
Yes, it should be made more obvious, but the packages on http://www.saurus.info/ are always up do date with the same commits as here.

**Comment by fgeek** [comment]
Ok. Do you plan to fix security vulnerabilities if those are found even the product is discontinued? If yes, for how long?
